README.md

```diff
@@ -51,5 +51,5 @@ If you want to participate in discussions about Thunderbird development, there a
 
 ### Chat
 
-If you want to ask questions in real-time about how to hack on Thunderbird, you can join our development chat channel at [https://chat.mozilla.org/\#/room/\#maildev:mozilla.org](https://chat.mozilla.org/#/room/#maildev:mozilla.org)
+If you want to ask questions in real-time about how to hack on Thunderbird, you can join our development chat channel at [#maildev:mozilla.org](https://matrix.to/#/#maildev:mozilla.org?via=mozilla.org&via=matrix.org&via=trancart.eu&web-instance%5Belement.io%5D=chat.mozilla.org)
```

Comment:
What is the advantage of matrix.to links instead of linking directly to the mozilla chat server? Do all `via` components need to be part of that link? What is "trancart.eu" ?

Reply:
Asking, because I have links to the chat server in the add-on section as well:
https://developer.thunderbird.net/add-ons/community

Reply:
`chat.mozilla.org` is _not_ the server, it is the element Matrix client that is hosted configured for mozilla there (as in, defaulting to the mozilla.org homeserver). Using a matrix.to link is future proof not only for once the `matrix:` protocol comes around, but also to support other clients, like mobile clients or eventually Thunderbird.

So if you're on mobile and click on a matrix.to link, it will suggest to open it with your favorite matrix app you have installed and let you join the room there. On desktop it will suggest you some clients, with a link to chat.mozilla.org at the top (that's what that web-instance param does).

When the mozilla matrix deployment was new matrix.to was not nearly as user friendly and the decision was to use chat.mozilla.org links for now, but those UX issues have now been largely fixed (especially when using the chat.mozilla.org preference).

Reply:
In regards to the vias, those are the home servers that the room is based in, so those are servers with accepted room aliases. They are used by some clients to figure out where to find the room, in case they know one of the servers already instead of trying to find mozilla.org. They aren't strictly required, but they are what will automatically be generated when you share the room from within the element client (and are also specified for the `matrix:` protocol eventually, since they give better robustness in case the primary home server that gives the room the name is down)

Reply:
Works as you say @freaktechnik - I'm going to approve it.